tracer/src/Datadog.Trace/Ci/Configuration/CIVisibilitySettings.cs

```diff
@@ -40,6 +40,8 @@ public CIVisibilitySettings(IConfigurationSource source)
             // Code coverage
             CodeCoverageEnabled = source?.GetBool(ConfigurationKeys.CIVisibility.CodeCoverage);
             CodeCoverageSnkFilePath = source?.GetString(ConfigurationKeys.CIVisibility.CodeCoverageSnkFile);
+            CodeCoveragePath = source?.GetString(ConfigurationKeys.CIVisibility.CodeCoveragePath);
+            CodeCoverageEnableJitOptimizations = source?.GetBool(ConfigurationKeys.CIVisibility.CodeCoverageEnableJitOptimizations) ?? true;
```

Comment:
this flag disables the jit optimization modifying this attribute in the target assembly: https://learn.microsoft.com/en-us/dotnet/api/system.diagnostics.debuggableattribute?view=net-7.0 . And yes this is a "just in case" flag.